Dz7/hw_7_task_2.js

```diff
@@ -0,0 +1,51 @@
+//Task 2
+//Написать функцию, которая принимает только предложение в качестве параметра и возвращает слово с наибольши кол-вом букв. 
+// Если таких слов несколько - возвращает их все
+
+
+function findLongestWord(Test_data) {
+    const new_strings = Test_data.split(" ") // Рабиваем строку на массив строк 
+    console.log(new_strings)
+
+    let longest_word = ""
+    const list_word = []
+
+    for (const item of new_strings) {
+        if (item.length > longest_word.length) {
+            longest_word = item // присваиваем значение
+            list_word.length = 0 // обнуляем массив, если нашли слово длиннее
+            list_word.push(item) // добавляем в массив новое значение
```

Comment:
Можно таким образом, только не забьіть `list_word` обьявить через `let`
```suggestion
            list_word = [item] // обнуляем массив, если нашли слово длиннее, добавляем в массив новое значение
```